types/artifacts.d.ts

```diff
@@ -179,6 +181,21 @@ declare global {
         params: {name: string; value: string}[];
       }
 
+      export interface IFrameElement {
+        /** The `id` attribute of the IFrame. */
+        id: string,
+        /** The `src` attribute of the IFrame. */
+        src: string,
+        /** The IFrame's ClientRect. @see https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect */
+        clientRect: ClientRect,
+        /** The visibility of the IFrame. */
+        isVisible: boolean,
```

Comment:
with `visibility` i think about visible within viewport or css visibility... this is neither.. what about a name like `hasSize` or... multiply width x height and call the result `pixelArea` and filter out the 0s later..

thoughts?

Reply:
Like the idea of pixelArea